test/e2e/common/volumes.go

```diff
@@ -394,6 +394,40 @@ var _ = framework.KubeDescribe("GCP Volumes", func() {
 	})
 
 	////////////////////////////////////////////////////////////////////////
+	// NFSv3
+	////////////////////////////////////////////////////////////////////////
+
+	framework.KubeDescribe("NFSv3", func() {
+		It("should be mountable for NFSv3", func() {
+			config := VolumeTestConfig{
+				namespace:   namespace.Name,
+				prefix:      "nfs",
+				serverImage: "gcr.io/google_containers/volume-nfs:0.8",
```

Comment:
Does it use a different server image then the v4 test? Maybe we should maintain two seperate nfs-server bianries in `gcr.io`, one for each version?